### PR TITLE
OSD-17611 : Adding information on Managed Scripts release cycles

### PR DESCRIPTION
**Pull request by MitaliBhalla**
As a part of [OSD-17611](https://issues.redhat.com//browse/OSD-17611), this PR intends to add information on Managed Scripts .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,3 +20,17 @@ the supported languages.
 ## `metadata.yaml`
 
 All `metadata.yaml` shall pass validation against `hack/metadata.schema.json` see [here](https://json-schema.org/) for more details
+
+## Release Cycle
+
+Managed Scripts follows the following release cycle :
```
Comment (supreeth7) [inline README.md:26]:
```suggestion
Managed Scripts has the following release cycle:
```

File: README.md [hunk 2]
```diff
@@ -20,3 +20,17 @@ the supported languages.
...
+
+### Staging:
+
+The staging environment will use the latest main branch of the managed-scripts github repo.
```
Comment (supreeth7) [inline README.md:30]:
```suggestion
The staging environment is pinned to consume the main branch of the managed-scripts repository.
```

File: README.md [hunk 3]
```diff
@@ -20,3 +20,17 @@ the supported languages.
...
+
+### Production:
+
+Every 3 weeks.
+
+For more information, please refer [Backplane Relase Cycles](https://source.redhat.com/groups/public/sre/wiki/backplane_user_documentation#isPasted5)
```
Comment (supreeth7) [inline README.md:36]:
```suggestion
For more information, please refer [Backplane Release Cycles](https://source.redhat.com/groups/public/sre/wiki/backplane_user_documentation)
```

File: README.md [hunk 4]
```diff
@@ -20,3 +20,17 @@ the supported languages.
...
+
+Managed Scripts follows the following release cycle :
+
+### Staging:
+
+The staging environment will use the latest main branch of the managed-scripts github repo.
+
+### Production:
+
+Every 3 weeks.
```
Comment (supreeth7) [inline README.md:34]:
```suggestion
Once every 3 weeks.
```

**Review by supreeth7 — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -20,3 +20,17 @@ the supported languages.
...
+
+### Staging:
+
+The staging environment is pinned to consume the main branch of the managed-scripts repository.
+
+### Production:
+
+Once every 3 weeks.
+
+For more information, please refer [Backplane Release Cycles](https://source.redhat.com/groups/public/sre/wiki/backplane_user_documentation)
```
Comment (bmeng) [inline README.md:36]: Same as the backplane-cli, I do not think we need to put the internal link in the public repo

Comment (MitaliBhalla) [reply README.md]: Updated

**Pull request merged**